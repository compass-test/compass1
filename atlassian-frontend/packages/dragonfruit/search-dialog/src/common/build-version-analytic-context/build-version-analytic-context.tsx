import React, { FunctionComponent } from 'react';
import { SearchDialogAnalyticsContext } from '../analytics';
import { BUILD_VERSION } from '../../buildVersion';

export const BuildVersionAnalyticContext: FunctionComponent<{}> = ({
  children,
}) => (
  <SearchDialogAnalyticsContext
    analyticContext={{ frontend: BUILD_VERSION }}
    nonPrivacySafeAnalyticContext={{}}
  >
    {children}
  </SearchDialogAnalyticsContext>
);
