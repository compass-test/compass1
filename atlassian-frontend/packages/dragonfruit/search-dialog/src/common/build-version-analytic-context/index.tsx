export { BuildVersionAnalyticContext } from './build-version-analytic-context';
