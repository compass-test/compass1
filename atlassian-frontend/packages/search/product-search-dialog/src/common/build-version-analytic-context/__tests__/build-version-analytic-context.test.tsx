import React from 'react';
import { shallow } from 'enzyme';
import { BuildVersionAnalyticContext } from '../build-version-analytic-context';

describe('<BuildVersionAnalyticContext />', () => {
  it('render matches snapshot', () => {
    const wrapper = shallow(
      <BuildVersionAnalyticContext>
        <div />
      </BuildVersionAnalyticContext>,
    );

    expect(wrapper.find('div').exists()).toBeTruthy();
    expect(wrapper).toMatchSnapshot();
  });
});
