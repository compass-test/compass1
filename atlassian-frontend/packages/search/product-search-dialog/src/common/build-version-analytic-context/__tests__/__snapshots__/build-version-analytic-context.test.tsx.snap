// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BuildVersionAnalyticContext /> render matches snapshot 1`] = `
<SearchDialogAnalyticsContext
  analyticContext={
    Object {
      "frontend": "@atlassian/product-search-dialog@__BUILD_VERSION__",
    }
  }
  nonPrivacySafeAnalyticContext={Object {}}
>
  <div />
</SearchDialogAnalyticsContext>
`;
